query-engine/query-engine-wasm/src/wasm/engine.rs

```diff
@@ -86,13 +91,17 @@ impl QueryEngine {
 
     /// Connect to the database, allow queries to be run.
     #[wasm_bindgen]
-    pub async fn connect(&self, trace: String) -> Result<(), wasm_bindgen::JsError> {
+    pub async fn connect(&self, trace: String, request_id: String) -> Result<(), wasm_bindgen::JsError> {
```

Comment:
We did we need an additional `request_id: String` parameter, again?

Reply:
to be able to fetch the spans associated with this request later